src/__swaps__/screens/Swap/hooks/useSwapTextStyles.ts

```diff
@@ -79,19 +60,30 @@ export function useSwapTextStyles({
   });
 
   const isInputZero = useDerivedValue(() => {
-    const isZero =
-      !internalSelectedInputAsset.value ||
-      (inputValues.value.inputAmount === 0 && inputMethod.value !== 'slider') ||
-      (inputMethod.value === 'slider' && equalWorklet(inputValues.value.inputAmount, 0));
-    return isZero;
+    const isInputAmountZero = inputValues.value.inputAmount === 0;
+    if (!internalSelectedInputAsset.value) return true;
+
+    if (inputMethod.value === 'slider' && equalWorklet(inputValues.value.inputAmount, 0)) return true;
+
+    if (inputMethod.value === 'inputNativeValue' && isInputAmountZero) {
+      return inputValues.value.inputNativeValue === 0;
```

Comment:
this adds an extra case to distinguish when we have typed into the `inputNativeValue` as the `inputMethod` and deleted all the characters, thus resulting in `inputNativeValue` equal to 0 (number)